worksheet_A/TerminalHacking.cpp

```diff
@@ -37,10 +85,49 @@ int main()
 	for each (std::string word in options)
 	{
 		std::cout << word << std::endl;
+		
 	}
 
-	// TODO: implement the rest of the game
```

Comment:
Thank you for removing the TODO comment after completing the task -- most people did not!